src/Popover/Popover.component.tsx

```diff
@@ -56,10 +56,10 @@ export class Popover extends React.Component<Props> {
     theme: Themes.defaultTheme,
   };
   public render() {
-    const { theme, arrow, children, ...props } = this.props;
+    const { theme, children, ...props } = this.props;
     const withArrowDistance = 10;
     const withoutArrowDistance = 9;
-    const distance = arrow ? withArrowDistance : withoutArrowDistance;
+    const distance = props.arrow ? withArrowDistance : withoutArrowDistance;
```

Comment:
It looks like there is not enough distance when the arrow is present.

Reply:
It's not clear in the design document, but I believe the tip of the arrow should be the same distance from the button as the top of the box in this design without the arrow. 

![Screen Shot 2019-04-11 at 11 55 05 AM](https://user-images.githubusercontent.com/2446153/55972150-f7709380-5c50-11e9-9ed3-784897e8d127.png)

Perhaps the distance should be an optional prop that could override this distance as well?

Reply:
actually, I see that distance is already an optional prop. 👍